pydantic/main.py

```diff
@@ -756,6 +756,20 @@ def update_forward_refs(cls, **localns: Any) -> None:
         for f in cls.__fields__.values():
             update_field_forward_refs(f, globalns=globalns, localns=localns)
 
+    @classmethod
+    def partial_model(cls: Type['Model']) -> Type['Model']:
```

Comment:
Adding again a new classmethod scares me a bit. Could we maybe have a more functional approach like `create_model`?
Something added in `pydantic.tools` perhaps?

Reply:
Haha, absolutely. I was just trying to avoid another import in any file that is doing a PATCH operation (in the context of FastAPI).